docs/labwc-config.5.scd

```diff
@@ -183,8 +183,10 @@ this is for compatibility with Openbox.
 
 *<core><allowTearing>* [yes|no]
 	Allow tearing to reduce input lag. Default is no.
-	This option requires setting the environment variable
-	WLR_DRM_NO_ATOMIC=1.
+	Use of this variable without setting the environment variable
+	WLR_DRM_NO_ATOMIC=1 is considered experimental at this time, and
+	may have minor issues if used in combination with hardware cursors,
+	though none were observed yet as a noticeable problem in testing.
 	*yes* allow tearing if requested by the active window.
```

Comment:
It seems the existing description was not properly edited when we added it. Please drop the final sentence, and maybe we can refine the description for clarity:

    Allow tearing, if requested by the active window, to reduce input lag.
    Default is no.
    
    Note: enabling this option with atomic mode setting is experimental. If
    you experience undesirable side effects when tearing is allowed,
    consider setting the environment variable WLR_DRM_NO_ATOMIC=1 when
    launching labwc.